app/api/v2/views/UserViews.py

```diff
@@ -0,0 +1,153 @@
+from ..models.UserModels import Order
+from validate_email import validate_email
+from flask_restful import Resource
+from flask import make_response, jsonify, request, abort
+from flask_jwt_extended import create_access_token
+from email.utils import parseaddr
+
+
+class DataParcel(Resource):
+    """Utilizes data from an order by either getting all 
+    data or posting new data"""
+
+    def post(self):
+        """Create an order"""
+        data = request.get_json()
+
+        par = Order()
+        par.create_order(data)
+
+        payload = {
+            "Status": "created",
+            "Data": data
+
+        }
+        result = make_response(jsonify(payload), 201)
+        result.content_type = 'application/json;charset=utf-8'
+        return result
+
+    def get(self):
+        data = request.get_json() or {}
+        """gets all orders made"""
+        par = Order()
+
+        all_orders = par.get_all()
+
+        result = make_response(jsonify({
+            "Status": "Ok",
+            "Orders": all_orders
+        }), 200)
+        result.content_type = 'application/json;charset=utf-8'
+        return result
+
+
+class SingleParcel(Resource):
+    """gets a single order"""
+
+    def get(self, order_id):
+        try:
+            order_id = int(order_id)
+        except Exception:
+            return {"Message": "Provide a valid order id",
+                    "Status": "bad request"}, 400
+
+        par = Order()
+        one_order = par.get_one_order(order_id)
+        if one_order is not None:
+            return make_response(jsonify(
+                {
+                    "Status": "Ok",
+                    "Orders": one_order
+                }))
+        return make_response(jsonify({
+            "Status": "Not Found"
+        }), 404)
+
+
+class CancelOrder(Resource):
+
+    def put(self, order_id):
+        """cancels an order that exists"""
+        data = request.get_json()
+        try:
+            order_id = int(order_id)
+        except Exception:
+            return make_response(jsonify({
+                "Message": "Provide a valid order id",
+                "Status": "bad request"
+            }), 400)
+        # if order_id is not None:
+        par = Order(data)
+        par.cancel_order(order_id)
+
+
+class RegisterUser(Resource):
+    """This class creates a new user"""
+
+    def post(self):
+
+        data = request.get_json() or {}
+
+        if 'username'not in data or 'email' not in data or\
```

Comment:
Move this validations to their own separate functions. This will make your view function cleaner and easier to debug.